doc/distrib/NodeHelpFiles/_review/Revit.Elements.Material.Color.md

```diff
@@ -0,0 +1,9 @@
+## In Depth
+`Material.Color` returns a Dynamo color that represents the material's cut pattern color (if available).
+
```

Comment:
What is the difference between Material.color node and Material.CutPatternColor node? They both seem to do the same thing.

Reply:
Added some additional text to distinguish as I believe material.color returns the default shading color